src/reporters/json.ts

```diff
@@ -54,9 +73,20 @@ class JSONReporter extends EmptyReporter {
   }
 
   onEnd() {
+    const suites = this.suite.suites.map(suite => this._serializeSuite(suite)).filter(s => s);
+    const commit = suites.length && suites[0].file ? getCommitInfo(path.dirname(suites[0].file)) : undefined;
     outputReport({
       config: this.config,
-      suites: this.suite.suites.map(suite => this._serializeSuite(suite)).filter(s => s),
+      commit,
+      runInfo: {
+        url: process.env['FOLIO_RUN_URL'] || '',
+        buildbotName: process.env['FOLIO_BUILDBOT_NAME'] || '',
```

Comment:
What happens when you have a test "A" that runs on different builders under different conditions? (browserName for example). How does parameters configuration relate to the build bot then? It is disregarded?